packages/react-scripts/config/webpackDevServer.config.js

```diff
@@ -60,4 +62,10 @@ module.exports = {
   https: protocol === 'https',
   host: host,
   overlay: false,
+  historyApiFallback: {
+    // Paths with dots should still use the history fallback.
+    // See https://github.com/facebookincubator/create-react-app/issues/387.
+    disableDotRule: true,
+  },
```

Comment:
Sure, but while I was setting up the minimal project I noticed that I couldn't reproduce it with that. So I decided to try again with my real project - and this time around I couldn't make the above happen. I don't really know what was going on, but it was definitely something strange. Sorry about that!